protocol-tokio/src/protocol/codec/message.rs

```diff
@@ -123,6 +126,10 @@ impl Message {
                 Data(lwcid, MessageType::MsgGetBlocks.encode_with(&gb))
             }
             Message::Block(lwcid, b) => Data(lwcid, MessageType::MsgBlock.encode_with(&b)),
+            Message::Subscribe(lwcid, keep_alive) => {
+                let keep_alive : u64 = if keep_alive { 43 } else { 42 };
```

Comment:
Is there other "source of truth" for these magic numbers other than this code?

Reply:
my theory is that the dev behind it chose 42 for false, and then followed up with 43 for true... or maybe it is because 6 by 9... unfortunately it is not easy to come by more regarding these...